2024-04-20 18:20:50,293 INFO    MainThread:1626514 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-04-20 18:20:50,293 INFO    MainThread:1626514 [wandb_setup.py:_flush():76] Configure stats pid to 1626514
2024-04-20 18:20:50,293 INFO    MainThread:1626514 [wandb_setup.py:_flush():76] Loading settings from /home/gpt/.config/wandb/settings
2024-04-20 18:20:50,293 INFO    MainThread:1626514 [wandb_setup.py:_flush():76] Loading settings from /home/gpt/grm/CLISA_Hydra/runs/2024-04-20/18-20-48_/wandb/settings
2024-04-20 18:20:50,293 INFO    MainThread:1626514 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-20 18:20:50,293 INFO    MainThread:1626514 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-04-20 18:20:50,293 WARNING MainThread:1626514 [wandb_setup.py:_flush():76] Could not save program above cwd: /home/gpt/grm/CLISA_Hydra/train_ext.py
2024-04-20 18:20:50,293 INFO    MainThread:1626514 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': None, 'program_abspath': '/home/gpt/grm/CLISA_Hydra/train_ext.py', 'program': '/home/gpt/grm/CLISA_Hydra/train_ext.py'}
2024-04-20 18:20:50,293 INFO    MainThread:1626514 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-20 18:20:50,293 INFO    MainThread:1626514 [wandb_init.py:_log_setup():521] Logging user logs to ./wandb/run-20240420_182050-4eg2m5ha/logs/debug.log
2024-04-20 18:20:50,293 INFO    MainThread:1626514 [wandb_init.py:_log_setup():522] Logging internal logs to ./wandb/run-20240420_182050-4eg2m5ha/logs/debug-internal.log
2024-04-20 18:20:50,294 INFO    MainThread:1626514 [wandb_init.py:init():561] calling init triggers
2024-04-20 18:20:50,294 INFO    MainThread:1626514 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {}
2024-04-20 18:20:50,294 INFO    MainThread:1626514 [wandb_init.py:init():611] starting backend
2024-04-20 18:20:50,294 INFO    MainThread:1626514 [wandb_init.py:init():615] setting up manager
2024-04-20 18:20:50,295 INFO    MainThread:1626514 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-04-20 18:20:50,297 INFO    MainThread:1626514 [wandb_init.py:init():623] backend started and connected
2024-04-20 18:20:50,298 INFO    MainThread:1626514 [wandb_init.py:init():715] updated telemetry
2024-04-20 18:20:50,299 INFO    MainThread:1626514 [wandb_init.py:init():748] communicating run to backend with 90.0 second timeout
2024-04-20 18:20:51,659 INFO    MainThread:1626514 [wandb_run.py:_on_init():2357] communicating current version
2024-04-20 18:20:51,951 INFO    MainThread:1626514 [wandb_run.py:_on_init():2366] got version response 
2024-04-20 18:20:51,951 INFO    MainThread:1626514 [wandb_init.py:init():799] starting run threads in backend
2024-04-20 18:20:54,679 WARNING MainThread:1626514 [wandb_init.py:init():1203] interrupted
Traceback (most recent call last):
  File "/home/gpt/anaconda3/envs/torch2/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 1181, in init
    run = wi.init()
          ^^^^^^^^^
  File "/home/gpt/anaconda3/envs/torch2/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 812, in init
    run_start_result = run_start_handle.wait(timeout=30)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gpt/anaconda3/envs/torch2/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gpt/anaconda3/envs/torch2/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gpt/anaconda3/envs/torch2/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gpt/anaconda3/envs/torch2/lib/python3.11/threading.py", line 622, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/gpt/anaconda3/envs/torch2/lib/python3.11/threading.py", line 324, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
